### PR TITLE
Replace logo.png by logo.svg

### PR DESCRIPTION
**Pull request by llrs-roche**
# Pull Request

Replace the logo.png by logo.svg to reduce package's size (This is relevant for CRAN submissions were packages cannot be generally >5Mb of source code and get flagged for CRAN reviewer if they exceed this number).

Important: the logo appears to be different: there is some green pattern on the SVG that is not on other logo formats. 

This package makes use of Github actions to check the branding, replaced there too. 

Fixes https://github.com/insightsengineering/nestdevs-tasks/issues/93

### CODE REVIEW

**Comment by llrs-roche** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by shajoezhu — APPROVED** [review]
lgtm! thanks @llrs-roche

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# `chevron`: Standard TLGs for clinical trials reporting <a href='https://insightsengineering.github.io/chevron/'><img src="man/figures/chevron.png" align="right" height="139" style="max-width: 100%; max-height: 139px;"/></a>
```
Comment (pawelru) [inline README.md:1]: hey @llrs-roche this does not work and need a follow-up PR
https://github.com/insightsengineering/chevron?tab=readme-ov-file
image

It has to be changed to `chevron.png` (or file renamed to `logo.png`)

I just picked this repo randomly. Please double check other logo updates.

Comment (llrs-roche) [reply README.md:1]: Thanks Pavel! I think I missed renaming the file to logo.svg or maybe we are stuck with png for github to show the logo as `usethis::use_logo` only uses PNG. I'll be checking this on all repos.

**Pull request merged**